tdrs-backend/manifest.yml

```diff
@@ -7,4 +7,4 @@ applications:
   docker:
     image: ((docker-backend))
   env:
-    AV_SCAN_URL: http://((cf-space))-clamav-rest.apps.internal:9000/scan
+    AV_SCAN_URL: http://tanf-prod-clamav-rest.apps.internal:9000/scan
```

Comment:
Do we only need this env var for local/circleci? If deployed, we have it hard-coded in `settings/cloudgov.py` now. Do we need to adjust `deploy_backend.sh`to no longer set that env var?

Reply:
No we don't need this. infact we will need to change this for non-prod instances to :

`AV_SCAN_URL: https://tdp-clamav-nginx-((env)).apps.internal:9000/scan`